code-studio/src/js/components/GridEditor.jsx

```diff
@@ -141,3 +139,9 @@ window.dashboard.GridEditor = module.exports = React.createClass({
     </div>);
   },
 });
+module.exports = GridEditor;
+
+// We put this on the dashboard namespace because GridEditor is used by _grid.html.haml
+// TODO _ exposed just a renderer?
+window.dashboard = window.dashboard || {};
+window.dashboard.GridEditor = GridEditor;
```

Comment:
I've been tempted to have a separate `levelbuilder` global for things in the levelbuilder bundle that we need to access this way.